_data/Diplomatie/Diplomatie.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Diplomatie
+institution: University Mcgill
+profile_pic: Diplomatie.jpg
```

Comment:
The name of this image and the one that you've uploaded do not match.

Reply:
I made the changes, thanks !